app.py

```
@@ -2865,6 +2950,32 @@ def recall_memories() -> Any:
 
+    # Emit SSE event for real-time monitoring
+    elapsed_ms = int((time.perf_counter() - query_start) * 1000)
+    result_count = 0
+    try:
+        # Response is either a tuple (response, status) or Response object
+        resp_data = response[0] if isinstance(response, tuple) else response
+        if hasattr(resp_data, "get_json"):
+            data = resp_data.get_json(silent=True) or {}
+            result_count = len(data.get("memories", []))
+    except Exception:
+        pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)